src/main/java/com/hypnoticocelot/appengine/helloworldservice/HelloWorldApplication.java

```diff
@@ -0,0 +1,48 @@
+//Copyright 2020 Google LLC
+//
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//
+//https://www.apache.org/licenses/LICENSE-2.0
+//
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+package com.hypnoticocelot.appengine.helloworldservice;
```

Comment:
Change the package to com.google.googleinterns.gscribe

Reply:
changed the package to  com.google.googleinterns.gscribe in the new commit.